airflow/utils/dag_processing.py

```diff
@@ -445,6 +446,7 @@ def file_path(self):
 
 
 DagParsingStat = namedtuple('DagParsingStat', ['file_paths', 'done', 'all_files_processed'])
+DagFileStat = namedtuple('DagFileStat', ['num_dags', 'import_errors', 'last_finish_time', 'last_duration', 'run_count'])
```

Comment:
Do you target to master branch? If yes, can you use a typed namedtuple here?
https://docs.python.org/3/library/typing.html#typing.NamedTuple

Reply:
I copied what was on the previous line mostly :) Will do here thought yeah.

(I will backport this to 1.10, but this PR is for master yes)